src/record/recordExpressionEvaluator/node/identifier/identifier.ts

```diff
@@ -0,0 +1,103 @@
+import { IdentifierExpression } from '../../../../expression'
+import { IdentifierEvaluator } from '../../../../expression/javascript/node/identifier'
+import { Node } from '../../../../node'
+import { NodeDef, NodeDefType } from '../../../../nodeDef'
+import { Records } from '../../../records'
+import { Survey, Surveys } from '../../../../survey'
+import { RecordExpressionContext } from '../../context'
+import { Objects } from '../../../../utils'
+import { RecordNodesFinder } from './recordNodesFinder'
```

Comment:
just call this NodesFinder (and fileName)

Reply:
ok